Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "spanner_client_rs"
+version = "0.1.0"
+authors = ["Donovan Preston <donovanpreston@gmail.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+futures = { version = "0.3", features = ["compat"] }
+googleapis-raw = { version = "0", path = "vendor/googleapis-raw" }
+protobuf = "2.12.0"
+grpcio = "0.5"
+
+[[bin]]
+name = "test"
```

Comment:
may want to use something a little less "test"y ;)

Reply:
> 51 files. Oof. Since this is a new check-in of a fork, I'm going to presume that a lot of the diffs are lost.

No, that's just the vendored checkin. If you look at the individual commits, it's a lot easier to read the diff.

Reply:
https://github.com/mozilla-services/spanner-client-rs/issues/3